BrutForce/Main/ViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  ViewController.swift
+//  BrutForce
+//
+//  Created by Дмитрий Виноградов on 08.03.2022.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    private var password: String = ""
+    
+    @IBOutlet weak var label: UILabel!
+    @IBOutlet weak var textField: UITextField!
+    @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
+    @IBOutlet weak var changeBGButton: UIButton!
+    @IBOutlet weak var generateButton: UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        label.text = "Here will be your password"
+        activityIndicator.isHidden = true
+    }
+
+    @IBAction func actionChangeBG(_ sender: Any) {
+        isBlack.toggle()
+        if view.backgroundColor == .black {
+            label.textColor = .white
+        } else {
+            label.textColor = .black
+        }
+    }
+    
+    @IBAction func ActionGenerate(_ sender: Any) {
+        if textField.hasText == false {
+            password = generateRandomPassword(3)
+            textField.isSecureTextEntry = true
+            DispatchQueue.main.async {
+                self.textField.text = self.password
+                self.activityIndicator.isHidden = false
+                self.activityIndicator.startAnimating()
+            }
+            
+            DispatchQueue.global().async {
+                self.bruteForce(passwordToUnlock: self.password)
+            }
+        } else {
+            password = textField.text ?? ""
+            textField.isSecureTextEntry = true
+            DispatchQueue.main.async {
+                self.textField.text = self.password
+                self.activityIndicator.isHidden = false
+                self.activityIndicator.startAnimating()
+            }
+            
+            DispatchQueue.global().async {
+                self.bruteForce(passwordToUnlock: self.password)
+            }
+        }
```

Comment:
Это что за дублирование кода?))
```suggestion
            password = textField.hasText ? (textField.text ?? "") : generateRandomPassword(3)
            textField.isSecureTextEntry = true
            DispatchQueue.main.async {
                self.textField.text = self.password
                self.activityIndicator.isHidden = false
                self.activityIndicator.startAnimating()
            }
            
            DispatchQueue.global().async {
                self.bruteForce(passwordToUnlock: self.password)
            }
```